addons/clones/addon.json

```
@@ -31,9 +31,9 @@
   ],
   "settings": [
     {
-      "id": "editoronly",
-      "name": "Only show in editor",
-      "default": true,
+      "id": "projectpage",
+      "name": "Also show on project page",
```

I don't think the word also is necessary here.
```suggestion
      "name": "Show on project page",
```